<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>767</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>767</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="button_layout">
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save Recipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="text">
         <string>New Recipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_recipe_button">
        <property name="text">
         <string>Modify Recipe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="recipes_dropdown"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="verticalFrame">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="recipe_name_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Recipe Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="recipe_name_input"/>
        </item>
        <item>
         <widget class="QLabel" name="ingredients_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Ingredients:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="ingredient_scroll_area">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="ingredient_scroll_content">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>284</width>
             <height>215</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="ingredients_layout_2"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_ingredient_button">
          <property name="text">
           <string>Add Ingredient </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="new_ingredient_input"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="steps_layout"/>
       </item>
       <item>
        <widget class="QPushButton" name="add_step_button">
         <property name="text">
          <string>Add Step</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
